internal/models/reservation_model.go

```diff
@@ -171,8 +171,10 @@ type AzureReservation struct {
 }
 
 type ReservationInstanceDetail struct {
-	PublicDNS  string `json:"public_dns"`
-	PublicIPv4 string `json:"public_ipv4"`
+	PublicDNS   string `json:"public_dns,omitempty"`
+	PublicIPv4  string `json:"public_ipv4,omitempty"`
+	PrivateIPv4 string `json:"private_ipv4,omitempty"`
+	PrivateIPv6 string `json:"private_ipv6,omitempty"`
 }
```

Comment:
This struct is used to store data in the JSONB column, added `omitempty` there is no reason to waste SQL resources with empty fields.

Reply:
It might be the reason why the IQE fails actually .... that now we  do not return instances at all? :thinking:

Reply:
Testing out without it: https://github.com/RHEnVision/provisioning-backend/pull/698

Reply:
Right, rebased.